packages/manager/modules/server-sidebar/src/web.constants.js

```diff
@@ -0,0 +1,378 @@
+  types: [
+    {
+      path: '/email/domain',
+      category: 'EMAIL_DOMAIN',
+      state: 'app.email.domain',
+      stateParams: ['productId'],
+      icon: 'ovh-font ovh-font-mail',
+      app: [WEB],
+    },
+    {
+      path: '/email/mxplan',
+      category: 'EMAIL_MXPLAN',
+      state: 'app.email.mxplan',
+      stateParams: ['productId'],
+      loadOnState: 'app.email.mxplan',
+      icon: 'ovh-font ovh-font-mail',
+      app: [WEB],
+    },
+  ],
+  icon: 'ovh-font ovh-font-mail',
+  app: [WEB],
+  regions: ['EU'],
+};
+
+
+export const MICROSOFT_CONFIG = {
+  id: 'microsoft',
+  children: [
+    {
+      id: 'exchange',
+      types: [
+        {
+          path: '/email/exchange',
+          icon: 'ms-Icon ms-Icon--ExchangeLogo',
+          loadOnState: 'app.microsoft.exchange',
+          stateParams: ['organizationId'],
+          app: [WEB],
+          types: [
+            {
+              path: '/email/exchange/:organizationId/service',
+              icon: 'ms-Icon ms-Icon--ExchangeLogo',
+              getState: ({ offer }) => {
+                const states = {
+                  provider: 'app.microsoft.exchange.provider',
+                  dedicated: 'app.microsoft.exchange.dedicated',
+                  dedicatedCluster: 'app.microsoft.exchange.dedicatedCluster',
+                  hosted: 'app.microsoft.exchange.hosted',
+                };
+
+                return states[offer];
+              },
+              stateParams: ['organization', 'productId'],
+              app: [WEB],
+            },
+          ],
+        },
+      ],
+      loadOnState: 'app.microsoft',
+      icon: 'ms-Icon ms-Icon--ExchangeLogo',
+      app: [WEB],
+    },
+    {
+      id: 'office',
+      types: [
+        {
+          path: '/license/office',
+          icon: 'ms-Icon ms-Icon--OfficeLogo',
+          state: 'app.microsoft.office.product',
+          stateParams: ['serviceName'],
+          app: [WEB],
+        },
+      ],
+      loadOnState: 'app.microsoft',
+      icon: 'ms-Icon ms-Icon--OfficeLogo',
+      app: [WEB],
+    },
+    {
+      id: 'sharepoint',
+      types: [
+        {
+          path: '/msServices/sharepoint',
+          icon: 'ms-Icon ms-Icon--SharepointLogo',
+          loadOnState: 'app.microsoft.sharepoint',
+          stateParams: ['organizationId'],
+          app: [WEB],
+          types: [
+            {
+              path: '/msServices/:organizationId/sharepoint',
+              icon: 'ms-Icon ms-Icon--SharepointLogo',
+              state: 'app.microsoft.sharepoint.product',
+              stateParams: ['organization', 'productId'],
+              app: [WEB],
+            },
+          ],
+        },
+      ],
+      loadOnState: 'app.microsoft',
+      icon: 'ms-Icon ms-Icon--ExchangeLogo',
+      app: [WEB],
+    },
+  ],
+  loadOnState: 'app.microsoft',
+  icon: 'ms-Icon ms-Icon--WindowsLogo',
+  app: [WEB],
+  regions: ['EU'],
+};
+
+/* export const MICROSOFT_CONFIG = {
```

Comment:
If not needed should probably be removed

Reply:
Thanks for pointing out this part. It will be addressed in separate pull request since a new release is required.